src/commands.c

```diff
@@ -5,179 +5,223 @@
  *
  * RedisRaft is licensed under the Redis Source Available License (RSAL).
  */
-/* ------------------------------------ Command Classification ------------------------------------ */
 
 #include <string.h>
 #include <ctype.h>
+
 #include "redisraft.h"
 
+static const CommandSpec commands[] = {
```

Comment:
Sorry, I should've mentioned this. This is part of config change, it is required to make it configurable on runtime rather than just on load time via module args.